commands/error.go

```diff
@@ -206,6 +206,22 @@ func IsCannotScaleBelowMinimumWorkersError(err error) bool {
 	return errgo.Cause(err) == cannotScaleBelowMinimumWorkersError
 }
 
+// user has selected an unknown AWS EC2 instance type
+var invalidAwsEc2InstanceTypeError = errgo.New("invalid AWS EC2 instance type selected")
```

Comment:
I see this new error, but I don't see it actually being used anywhere. Is there any validation on the instance type string? (besides conflicting flag validation that I see in this PR)

Reply:
Good catch! That's a remainder from when i had validation implemented.

Reply:
There is server side validation luckily for this value, so I'm fine if gsctl doesn't validate :)